recipes/requestsexceptions/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{%set name = "requestsexceptions" %}
+{%set version = "1.1.3" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "d678b872f51f76d875e00e6667f4ddbf013b3a99490ae5fe07cf3e4f846e283e" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
```

Comment:
I was debating this, but I think that keeping to this format makes it relatively easier to stub out new recipes. Long-term, the best answer might be to make a small script to help with the stubbing instead - something less heavy than `skeleton` but that will fill out the sha256 values automatically.